src/CommunityToolkit/Xamarin.CommunityToolkit/Views/ViewSwitcher/TextSwitcher.shared.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class TextSwitcher : Label
```

Comment:
Shouldn't this class implement the `IViewSwitcher`? I'm asking because this approach is similar to the `ITextElement` on Label control.
references:
https://github.com/xamarin/xamarin.forms/blob/5.0.0/Xamarin.Forms.Core/Label.cs#L15
https://github.com/xamarin/xamarin.forms/blob/5.0.0/Xamarin.Forms.Core/ITextElement.cs
https://github.com/xamarin/Xamarin.Forms/blob/ec1520e32439f9cb2929883bacf3db138165bbb3/Xamarin.Forms.Core/TextElement.cs#L3